config/locales/en.yml

```diff
@@ -1146,7 +1146,7 @@ en:
 
   transactions:
     in_review: Transactions in Review
-    none: You have no transactions in review, or any recently reviewed.
+    none_recently_reviewed: No transactions in review were found.
```

Comment:
If this is the message for no transactions currently in review, then the key doesn't feel right.